files/build_templates/init_cfg.json.j2

```diff
@@ -4,7 +4,8 @@
             "buffer_model": {% if default_buffer_model == "dynamic" %}"dynamic"{% else %}"traditional"{% endif %},
             {%- if include_p4rt == "y" %}"synchronous_mode":"enable",{% endif %}
             "default_bgp_status": {% if shutdown_bgp_on_start == "y" %}"down"{% else %}"up"{% endif %},
-            "default_pfcwd_status": {% if enable_pfcwd_on_start == "y" %}"enable"{% else %}"disable"{% endif %}
+            "default_pfcwd_status": {% if enable_pfcwd_on_start == "y" %}"enable"{% else %}"disable"{% endif %},
+            "yang_config_validation": {% if enable_yang_config_validation == "y" %}"enable"{% else %}"disable"{% endif %}
```

Comment:
>yang_config_validation [](http://example.com/codeflow?start=13&length=22)

All other parameters are "default_*", is your intention is the same, like "default_enable_..."?